service/src/main/java/bio/terra/pipelines/app/controller/PipelineRunsApiController.java

```diff
@@ -144,6 +144,17 @@ public ResponseEntity<ApiAsyncPipelineRunResponse> startPipelineRun(
         jobId,
         userId);
 
+    // we don't want to start a pipeline whose input data would have been deleted because
+    // of the TTL on user data.
+    if (pipelineRunBeforeStart
+        .getCreated()
+        .plus(pipelinesCommonConfiguration.getUserDataTtlDays(), ChronoUnit.DAYS)
+        .isBefore(Instant.now())) {
+      throw new BadRequestException(
+          "Pipeline run was prepared more than %s days ago, it can not be started"
```

Comment:
nit
```suggestion
          "Pipeline run was prepared more than %s days ago; it cannot be started"
```

Reply:
esl :P (i also changed another message i wrote in a past PR that was similarly not right)